pkg/auth/auth_assisted_service_mock_test.go

```diff
@@ -473,7 +473,15 @@ func (f fakeInventory) V2GetClusterDefaultConfig(ctx context.Context, params ins
 }
 
 func (f fakeInventory) V2DownloadClusterLogs(ctx context.Context, params installer.V2DownloadClusterLogsParams) middleware.Responder {
-	return installer.NewV2DownloadClusterLogsOK()
+	file, err := ioutil.TempFile("/tmp", "test.file")
+	if err != nil {
+		return installer.NewV2DownloadClusterLogsInternalServerError().WithPayload(
+			common.GenerateError(http.StatusInternalServerError, err))
+	}
+	return filemiddleware.NewResponder(
+		installer.NewV2DownloadClusterLogsOK().WithPayload(io.ReadCloser(file)),
```

Comment:
You can do this with a [`strings.Reader`](https://pkg.go.dev/strings@go1.17.7#Reader) then we won't have to worry about cleaning this file up (which I think is a problem here).

Reply:
Done.

Reply:
This isn't resolved. You're still using a file here.

Reply:
sorry. fixed now.